🗂️ hub.md

```diff
@@ -21,6 +21,17 @@ publish: true
 -  [[06 - Inbox/🗂️ 06 - Inbox|🗂️ 06 - Inbox]]
 -  [[00 - Start here|00 - Start here]]
 -  [[CONTRIBUTING|CONTRIBUTING]]
+-  [[Editing notes using the github.dev editor|Editing notes using the github.dev editor]]
+-  [[github compare forks.png|github compare forks.png]]
+-  [[github create pull request.png|github create pull request.png]]
+-  [[github fill out PR template.png|github fill out PR template.png]]
+-  [[github.dev editor commit changes.png|github.dev editor commit changes.png]]
+-  [[github.dev editor create branch.png|github.dev editor create branch.png]]
+-  [[github.dev editor stage changes.png|github.dev editor stage changes.png]]
+-  [[github.dev editor version control.png|github.dev editor version control.png]]
+-  [[github.dev editor.png|github.dev editor.png]]
+-  [[github.dev fork view.png|github.dev fork view.png]]
+-  [[github.dev switch to fork.png|github.dev switch to fork.png]]
```

Comment:
These images files should be moved to the images directory, so they are not listed in the top-level MOC.

I'll close this PR, move the files, and then re-run the action.